### DISCUSSION TITLE
HOGWARTS: Weird email message in form

### DISCUSSION THREAD

**Discussion by TaylorHardaway** [nss-evening-cohort-14/evening-client-side#85] [Help Me!] [answered]
Taylor Hardaway

I don't see any other help tickets on this.

I got a form from bootstrap that I modified the wording on, but it was the best one I could find. When I type in a name to try out what I have it asks for an email. I did put the e.preventDefault() method in there, but perhaps I have something coded wrong.

const getHouse = (e) => {
  e.preventDefault();

  const name = document.querySelector("#name").value;

  const obj = {
    name,
    house,
    id,

    
  };
  students.push(obj);
  studentsForm(students);
  document.querySelector('form').reset()
}
http://reusableforms.com/d/o5/html5-contact-form-send-email
https://stackoverflow.com/questions/43918751/how-can-i-send-form-contact-to-email-using-javascript
https://stackoverflow.com/questions/60272725/email-validation-using-javascript-with-a-custom-message-below-the-input-field
(everything I find is the opposite of what I want)
I try continuing to code hoping that I just have something else wrong.
https://github.com/TaylorHardaway/sorting-hat/blob/First-Steps/main.js

**Comment by drteresavasquez** [comment]
Your input has a type of email...change that.

**Comment by ajawashington** [comment, marked as answer]
https://github.com/TaylorHardaway/sorting-hat/blob/First-Steps/main.js#L49-L50

You have the "type=email" on both your label and input field.

**Comment by TaylorHardaway** [comment]
Yep, so I changed the type to "text" and made a more specific label to what I am doing, thanks.
